sBTC SIP

```diff
@@ -0,0 +1,166 @@
+- sBTC Bootstrap Signers are separate from Stacks Signers. In a subsequent sBTC release, and accompanying Stacks network hard fork, Stacks signers will be required to become sBTC signers. The bootstrapping period uses a separate subset of Stacks signers to secure the sBTC protocol and Signer operations are not explicitly linked with slashing of rewards.
+- sBTC deposits will be triggered via an API call. During the bootstrap phase sBTC deposits must be initiated via an API call to alert the signers to the presence of a deposit. After the bootstrapping phase users will have the option to alert the Signers of a deposit via a Stacks contract call, but the API will remain as a fee-less option to communicate with the Signers.
+
+These changes enable sBTC to be launched earlier and with less risk than the long term decentralized approach. The sBTC release with an open and dynamic signer set is expected to be released early 2025, which would enshrine the sBTC protocol into Stacks consensus.
+
+## Specification
+
+### Deposits (Minting sBTC)
+An API call referencing a Bitcoin transaction initiates the deposit process that can be completed within a single Bitcoin block. The API is separate from the sBTC Bootstrap Signers but will need access to the sBTC Bootstrap Signers to let them know which transactions to pick up.
+
+**API:** The API relays deposit information to the sBTC Bootstrap Signers. The API reduces the cost for deposit on rejection, reduces the scope of sBTC, and makes the sBTC system easier to service.
+
+**Bitcoin Deposit Information:** The deposit will need to pay to taproot and be spendable by a consensus threshold of the signers. The API will need to know all of the taproot spend paths and the spend paths will need to fit a specific format that cannot be clawed back in the short term in order for the Signers to consider approving it.
+
+#### Deposit Flow
+The main steps of the sBTC Deposit flow are as follows:
+1. **Deposit request:** A bitcoin holder creates a transaction on Bitcoin.
+    - The deposit transaction contains a UTXO (deposit UTXO) spendable by sBTC Signers, with an OP_DROP payload.
+    - The payload contains the recipient address of the sBTC among other relevant info for the deposit.
+    - The relevant info could contain fee suggestion or max_fee.
+2. **Proof of deposit:** The bitcoin holder submits a proof of deposit on Stacks by invoking the Signer API.
+3. **Deposit accept:**
+4. **Deposit redeem:** The sBTC Signers redeem the deposit by consuming the deposit UTXO, consolidating it into the sBTC UTXO.
```

Comment:
Who pays for the sBTC signers' transaction fee for redeeming it?

Do the signers even need to redeem it at all?  For example, could they instead just spend it as part of processing an sBTC withdrawal (and thereby forward the Bitcoin transaction fee to the withdrawing party)?

Reply:
The deposit redemption is paid for by the deposit transaction. The deposit format includes some "parameters" within it and one of those is the max_fee that they're willing to pay to facilitate the consolidation.

The LLD for this is here: https://docs.google.com/document/d/1iQ-OCYqQL31JVb_fhBoHYj2Ay0drBfmypa6HDZa9lVU/edit#heading=h.cjaifsbl2qsc

Reply:
I read that document, and it describes the mechanics of setting a `max-fee`, but it opens more questions.  What happens if `max-fee` is too small?  Or too big?  Or not to the signer's liking?  Can consolidation get stuck indefinitely due to a low fee, or an inability to RBF?